uptane/clients/primary.py

```diff
@@ -564,22 +625,47 @@ def primary_update_cycle(self):
 
       tuf.formats.TARGETFILE_SCHEMA.check_match(target) # redundant, defensive
 
-      if 'custom' not in target['fileinfo'] or \
-          'ecu_serial' not in target['fileinfo']['custom']:
-        raise uptane.Error('Director repo failed to include an ECU Serial for '
+      if 'custom' not in target['fileinfo']:
+        raise uptane.Error('Director repo failed to include the custom field in '
             'a target. Target metadata was: ' + repr(target))
+      else:
+        for custom_field in ['ecu_serial', 'hardware_id', 'release_counter']:
+          if custom_field not in targetinfo['fileinfo']['custom']:
+            raise uptane.Error(
+              "Director repo failed to include an {} for a target. Target metadata was: {}".format(custom_field, repr(target)))
```

Comment:
I completely agree. I did this to make checks in the update cycle for the primary (I thought the primary updates worked). 

Is it fine if I keep this till we decide on how to implement the primary's updates? Or would you prefer if I removed these anyway?